src/pages/index.tsx

```diff
@@ -1,8 +1,22 @@
-import { Inter } from "next/font/google";
-import { MainLayout } from "@/components/layout/MainLayout";
-const inter = Inter({ subsets: ["latin"] });
-
+import { Button, Input } from '@/components/common'
+import { MainLayout } from '@/components/layout/MainLayout'
 
 export default function Home() {
-  return <MainLayout>hello</MainLayout>;
-};
+  return (
+    <MainLayout>
+      <div className='mx-auto my-10 flex w-80 flex-wrap justify-center gap-4'>
+        <p className='w-fit rounded-lg bg-primary p-2 text-xl'>STEP1</p>
+        <p className='my-auto flex-wrap text-xl underline'>イベント名を入力</p>
+        <Input></Input>
```

Comment:
inputはchildrenを持たないので一つにまとめてしまって大丈夫
```suggestion
        <Input />
```

Reply:
[c45fbde]　(https://github.com/NUTFes/chosei-chan/pull/28/commits/c45fbde3ad81436eca3f37e76b46aae22880aed6)修正しました。